### PR TITLE
New package: qupzilla-1.8.6

### LINKED ISSUE #1968: Package request: QupZilla
QupZilla

http://www.qupzilla.com QupZilla is a new and very fast lightweight Qt WebKit Browser.

More info at: http://qt-apps.org/content/show.php/QupZilla?content=145693

Would be nice to have an .xbps for it since it's quite long/tricky to build it yourself.

Thank you...

### PR DESCRIPTION
**Pull request by pullmoll**
Closes #1968
Should be merged after #1957

### CODE REVIEW

File: srcpkgs/qupzilla/template [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+license="GPL-3"
+homepage="http://www.qupzilla.com/"
+distfiles="https://github.com/QupZilla/${pkgname}/releases/download/v${version}/QupZilla-${version}.tar.xz"
+checksum=aecda29f8994c9dc4d6b5f76ce5b1ba947272a0d5a40b5d83a54023dd5d7a6a5
+
+pre_build() {
+	qmake
+}
+do_install() {
+	make INSTALL_ROOT=${DESTDIR} install
```
Comment (beefcurtains) [inline srcpkgs/qupzilla/template]: Hm, ``make_install_args`` not good enough?

File: srcpkgs/qupzilla/template [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Template file for 'qupzilla'
+pkgname=qupzilla
+version=1.8.6
+revision=1
+wrksrc=QupZilla-${version}
+build_style=gnu-makefile
+hostmakedepends="pkg-config"
+makedepends="qt5-devel qt5-script-devel qt5-tools-devel qt5-webkit-devel"
+short_desc="Lighhweight multiplatform browser"
```
Comment (beefcurtains) [inline srcpkgs/qupzilla/template]: Lightweight

File: srcpkgs/qupzilla/template (hunk 1 above)
Comment (pullmoll) [inline srcpkgs/qupzilla/template]: It probably is. I hadn't thought of that.

Comment (pullmoll) [inline srcpkgs/qupzilla/template]: Hmm.. it installs into the real root (/) now. Let me try to escape the `$`...

**Comment by beefcurtains** [comment]
:+1:

**Comment by xtraeme** [comment]
:+1:

**Pull request merged**